lighthouse-core/audits/time-to-interactive.js

```diff
@@ -79,17 +79,22 @@ class TTIMetric extends Audit {
       const fMPts = timings.fMPfull + timings.navStart;
 
       // look at speedline results for 85% starting at FMP
-      let eightyFivePctVC = artifacts.Speedline.frames.find(frame => {
-        return frame.getTimeStamp() >= fMPts && frame.getProgress() >= 85;
-      });
+      let eightyFivePctVC;
+      let visuallyReadyTiming = 0;
 
-      // Check to avoid closure compiler null dereferencing errors
-      if (eightyFivePctVC === undefined) {
-        eightyFivePctVC = 0;
-      }
+      if (artifacts.Speedline.frames) {
+        eightyFivePctVC = artifacts.Speedline.frames.find(frame => {
+          return frame.getTimeStamp() >= fMPts && frame.getProgress() >= 85;
+        });
 
-      // TODO CHECK these units are the same
-      const visuallyReadyTiming = (eightyFivePctVC.getTimeStamp() - timings.navStart) || 0;
+        // Check to avoid closure compiler null dereferencing errors
+        if (eightyFivePctVC === undefined) {
+          eightyFivePctVC = 0;
+        }
+
+        // TODO CHECK these units are the same
+        visuallyReadyTiming = eightyFivePctVC.getTimeStamp() - timings.navStart || 0;
+      }
```

Comment:
can do
```
if (eightyFivePctVC === undefined) {
  visuallyReadyTiming = 0;
} else {
  visuallyReadyTiming = eightyFivePctVC.getTimeStamp() - timings.navStart;
}
```
(and move `eightyFivePctVC` inside the `if (artifacts.Speedline.frames)` block)